src/screens/LegalRights/VideoPage/styles.ts

```diff
@@ -0,0 +1,35 @@
+import { StyleSheet } from 'react-native';
+
+export const styles = StyleSheet.create({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    marginTop: 10,
+    padding: 30,
+  },
+  video: {
+    width: '100%',
+    height: 630,
+  },
+  buttonContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    paddingVertical: 25,
+  },
+  captionButtons: {
+    borderRadius: 10,
+    width: 180,
+    height: 50,
+    borderWidth: 1,
+    backgroundColor: '#D9D9D9',
+    borderColor: '#D9D9D9',
+    shadowColor: 'black',
+    shadowOpacity: 0.05,
```

Comment:
not super sure, but pretty sure these 2 lines are unnecessary for now since we don't have any shadow